.gitignore

```diff
@@ -1,3 +1,33 @@
 node_modules
 build/**/*.ts
 !build/**/*.build.js
+build/content-viewer.build.js
```

Comment:
Can you remove all the changes from .gitignore? We don't actually want to ignore these files, it's just that we only need to commit the built bundles when we're releasing a new version, otherwise we can leave them out.

Reply:
Also I see changes to the files in `build/`, can you remove all of those changes too? Diffs of built bundles bloat the repo and we don't need them yet.

Normally what I do is just selectively add individual files the commits I'm making, so I can avoid adding these bundles.